vslib/src/SwitchStateBase.cpp

```diff
@@ -1793,6 +1800,10 @@ sai_status_t SwitchStateBase::refresh_read_only(
             case SAI_SWITCH_ATTR_NUMBER_OF_SYSTEM_PORTS:
             case SAI_SWITCH_ATTR_SYSTEM_PORT_LIST:
                 return refresh_system_port_list(meta);
+
+            case SAI_SWITCH_ATTR_NUMBER_OF_FABRIC_PORTS:
+            case SAI_SWITCH_ATTR_FABRIC_PORT_LIST:
+                return SAI_STATUS_SUCCESS;
```

Comment:
is this happening on each brcm chip ?

Reply:
This should be on both NPU and fabric asics.

Can I have a context why you asked this? Maybe I missed something?

Reply:
in number of fabric port can change on runtime, then thosevalues must be refreshed in similar way as other ports

Reply:
Different from front panel ports which could be disabled/enabled in runtime, currently fabric ports don't. All fabric ports are brought up at chip init, and then they are always there. But I could have `refresh_fabric_port_list` in case in future we want to enable/disable fabric ports, but of course currently `refresh_fabric_port_list` does nothing and just returns `SAI_STATUS_SUCCESS`. Does it make sense to you?